openshift/tests-extension/test/olmv1-singleownnamespace.go

```diff
@@ -359,6 +337,29 @@ var _ = Describe("[sig-olmv1][OCPFeatureGate:NewOLMOwnSingleNamespace] OLMv1 ope
 					activeNamespaces[watchNamespace] = struct{}{}
 				}
 
+				// Ensure unique names per scenario
+				crdSuffix := rand.String(4)
+				packageName := fmt.Sprintf("singleown-operator-both-%s", crdSuffix)
+				By(fmt.Sprintf("building singleown operator assets for %s scenario: image=%s, CRD suffix=%s, package=%s", sc.label, singleownImage, crdSuffix, packageName))
+
+				replacements := map[string]string{
+					"{{ TEST-BUNDLE }}":                        "",
+					"{{ NAMESPACE }}":                          "",
+					"{{ TEST-CONTROLLER }}":                    singleownImage,
+					"{{ CRD-SUFFIX }}":                         crdSuffix, // Unique CRD suffix per scenario
+					"{{ PACKAGE-NAME }}":                       packageName,
+					"webhook-operator-webhooktest-admin-role":  fmt.Sprintf("webhook-operator-webhooktest-admin-role-%s", crdSuffix),
```

Comment:
FYI
```go
					"webhook-operator-webhooktest-admin-role":  fmt.Sprintf("webhook-operator-webhooktest-admin-role-%s", crdSuffix),
					"webhook-operator-webhooktest-editor-role": fmt.Sprintf("webhook-operator-webhooktest-editor-role-%s", crdSuffix),
					"webhook-operator-webhooktest-viewer-role": fmt.Sprintf("webhook-operator-webhooktest-viewer-role-%s", crdSuffix),
					"webhook-operator-metrics-reader":          fmt.Sprintf("webhook-operator-metrics-reader-%s", crdSuffix),
```

although I do not think it is necessary (actually it is not used), it will not impact the case execution.